spec/index.html

```diff
@@ -1786,6 +1905,7 @@ <h3>RDF Term Constructors</h3>
       <tr id="handle-PNAME_NS"                        ><td style="text-align:left;" rowspan="2"><a class="type string"       href="#grammar-production-PNAME_NS"                        >PNAME_NS                        </a></td><td><a href="#prefix">                              prefix       </a></td><td>When used in a <a href="#grammar-production-prefixID"><code>prefixID</code></a> or <a href="#grammar-production-sparqlPrefix"><code>sparqlPrefix</code></a> production, the <code>prefix</code> is the potentially empty <a data-cite="RDF12-CONCEPTS#dfn-rdf-string">RDF string</a> matching the first argument of the rule is a key into the <a href="#namespaces">namespaces map</a> into which the expanded second argument is stored for future lookup.</td></tr>
       <tr id="handle-PNAME_NS2"                       >                                                                                                                                                                           <td><a data-cite="RDF12-CONCEPTS#dfn-iri">          IRI          </a></td><td>When used in a <a href="#grammar-production-PrefixedName"><code>PrefixedName</code></a> production; the <a href="#namespaces">namespaces map</a> MUST have a corresponding <code>namespace</code>, which forms the <a data-cite="RDF12-CONCEPTS#dfn-rdf-string">RDF string</a> of the IRI.</td></tr>
       <tr id="handle-PNAME_LN"                        ><td style="text-align:left;"            ><a class="type IRI"          href="#grammar-production-PNAME_LN"                        >PNAME_LN                        </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-iri">          IRI          </a></td><td>A potentially empty <a href="#prefix">prefix</a> is identified by the first sequence, <a href="#grammar-production-PNAME_NS"><code>PNAME_NS</code></a>. The <a href="#namespaces">namespaces map</a> <em class="rfc2119">MUST</em> have a corresponding <code>namespace</code>. The <a data-cite="RDF12-CONCEPTS#dfn-rdf-string">RDF string</a> of the IRI is formed by unescaping the <a data-lt="reserved character escape sequence">reserved characters</a> in the second argument, <a href="#grammar-production-PN_LOCAL"><code>PN_LOCAL</code></a>, and concatenating this onto the <code>namespace</code>.</td></tr>
+      <tr id="handle-VERSION_SPECIFIER"               ><td style="text-align:left;"            ><a class="type string"       href="#grammar-production-VERSION_SPECIFIER"               >VERSION_SPECIFIER               </a></td><td><a data-cite="RDF12-CONCEPTS#dfn-literal">      literal      </a></td><td>The characters between the <a href="#cp-quotation-mark"><code title="quotation mark">&quot;</code></a>s are taken to form the decimal value of |curVersion|.</td></tr>
```

Comment:
I think there's a tension here between using a literal form (`"1.2"`) in the surface syntax and a decimal in the parser state definition. If it's really just a decimal, why not a decimal in the surface syntax? On the other hand, I think it might be useful to allow more than just decimals. For example:
* `@version "1.1-rdf-star-cg"` for data created for a system implementing the CG's version of rdf-star which might be syntactically valid RDF 1.2, but doesn't match the semantics
* `@version "1.3-dev` for in-development features that don't yet have agreed upon semantics, but allowing safe, cross-implementation interop experimentation

Reply:
Also, if we want to support a 1.2-subset allowing langdirs but no triple terms… that would probably be more than just a decimal.

Reply:
Good argument that |curVersiob| should remain a string. But, other values are hypothetical are hypothetical at this point.

Reply:
+1 on `xsd:string`

Reply:
Instead of allowing any string in quotes, what about allowing any sequence of ascii digits and lower case leter separated by `.` or `-`? The grammar would be something like `[0-9a-z]+ ([.-] [0-9a-z]+)*`

This way one can write `@version 1.2 .` or `@version 1.1-rdf-star-cg .` or `@version 1.3-alpha.4 .` 
I find it slightly nicer than strings in quotes while still allowing a big range of values.